storage/migrations/schema/external/from0.ts

```diff
@@ -1,7 +1,10 @@
 import { Upgrade } from "remultiform/database";
 import ExternalDatabaseSchema from "../../../ExternalDatabaseSchema";
 
-export default (upgrade: Upgrade<ExternalDatabaseSchema["schema"]>): void => {
+const upgradeSchema = (
+  upgrade: Upgrade<ExternalDatabaseSchema["schema"]>
+): void => {
```

Comment:
just out of curiosity- why this change?

Reply:
is it just a refactor?

Reply:
no, this was highlighted when I updated next.js, here's the error 
![image](https://user-images.githubusercontent.com/64883/89908296-9e6a9400-dbe5-11ea-9f04-2b440d7da07b.png)